PornHub won't rip

* Ripme version: 1.7.84
* Java version: <!-- (output of `java -version`) --> Oracle Java SE 8 Update 211 x64
* Operating system: <!-- (if Windows, output of `ver` or `winver`) --> Window$ 10.0.17134.706
<!-- Please do not link to content featuring underage characters even if the characters are drawn.
 These works are still illegal in many places including much of America -->
* Exact URL you were trying to rip when the problem occurred: [NSFW] https://www.pornhub.com/view_video.php?viewkey=ph5c8061033ff50
* Please include any additional information about how to reproduce the problem:
  - RipMe output:
```
Rip complete, saved to D:\_DOWNLOADS_\RipMe\rips\videos
java.net.MalformedURLException: no protocol: 
```

  - `.\ripme.log`:
```
2019-05-27 18:39:28,350 INFO  ui.MainWindow .loadHistory() Loading history from D:\_DOWNLOADS_\RipMe\history.json
2019-05-27 18:39:28,444 DEBUG ui.UpdateUtils .updateProgramGUI() Retrieving https://raw.githubusercontent.com/ripmeapp/ripme/master/ripme.json
2019-05-27 18:39:29,964 DEBUG ui.UpdateUtils .updateProgramGUI() This version (1.7.84) is the same or newer than the website's version (1.7.84)
2019-05-27 18:39:29,966 DEBUG ui.UpdateUtils .updateProgramGUI() Running latest version: 1.7.84
2019-05-27 18:39:30,244 DEBUG ripper.AbstractRipper .getRipper() Found video ripper: com.rarchives.ripme.ripper.rippers.video.PornhubRipper
2019-05-27 18:39:33,973 DEBUG ripper.AbstractRipper .getRipper() Found video ripper: com.rarchives.ripme.ripper.rippers.video.PornhubRipper
2019-05-27 18:39:33,973 DEBUG ripper.AbstractRipper .setWorkingDir() Set working directory to: D:\_DOWNLOADS_\RipMe\rips\videos
2019-05-27 18:39:38,788 DEBUG ripper.AbstractRipper .getRipper() Found video ripper: com.rarchives.ripme.ripper.rippers.video.PornhubRipper
```

  - `.\rips\videos\log.txt`:
```
2019-05-27 18:39:33,975 DEBUG ripper.DownloadThreadPool .initialize() Initializing Main thread pool with 1 threads
2019-05-27 18:39:33,981 INFO  ripper.AbstractRipper .rip()     Retrieving https://www.pornhub.com/view_video.php?viewkey=ph5c8061033ff50
2019-05-27 18:39:35,274 ERROR ripper.AbstractRipper .rip() Error while retrieving video URL at https://www.pornhub.com/view_video.php?viewkey=ph5c8061033ff50
java.net.MalformedURLException: no protocol: 
	at java.net.URL.<init>(Unknown Source)
	at java.net.URL.<init>(Unknown Source)
	at java.net.URL.<init>(Unknown Source)
	at com.rarchives.ripme.ripper.rippers.video.PornhubRipper.rip(PornhubRipper.java:81)
	at com.rarchives.ripme.ripper.AbstractRipper.run(AbstractRipper.java:616)
	at java.lang.Thread.run(Unknown Source)
2019-05-27 18:39:35,276 ERROR ripper.AbstractRipper .run() Got exception while running ripper:
java.io.IOException: java.net.MalformedURLException: no protocol: 
	at com.rarchives.ripme.ripper.rippers.video.PornhubRipper.rip(PornhubRipper.java:87)
	at com.rarchives.ripme.ripper.AbstractRipper.run(AbstractRipper.java:616)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.net.MalformedURLException: no protocol: 
	at java.net.URL.<init>(Unknown Source)
	at java.net.URL.<init>(Unknown Source)
	at java.net.URL.<init>(Unknown Source)
	at com.rarchives.ripme.ripper.rippers.video.PornhubRipper.rip(PornhubRipper.java:81)
	... 2 more
2019-05-27 18:39:35,277 DEBUG ripper.AbstractRipper .waitForThreads() Waiting for threads to finish
2019-05-27 18:39:35,277 INFO  ripper.AbstractRipper .checkIfComplete()    Rip completed!
```

  - **I recall _youtube-dl_ changed accordingly, so that might be a good place to look since it works if you don't wanna figure it out yourself.**
## Expected Behavior
The ripping fails a few seconds after hitting 'Rip' button.

## Actual Behavior
Rip normally.

--- a/src/main/java/com/rarchives/ripme/ripper/rippers/video/PornhubRipper.java
+++ b/src/main/java/com/rarchives/ripme/ripper/rippers/video/PornhubRipper.java
@@ -69,10 +69,12 @@ public class PornhubRipper extends VideoRipper {
                 int bestQual = 0;
                 for (int i = 0; i < mediaDef.length(); i++) {
                     JSONObject e = (JSONObject) mediaDef.get(i);
-                    int quality = Integer.parseInt((String)e.get("quality"));
-                    if (quality > bestQual) {
-                        bestQual = quality;
-                        vidUrl = (String)e.get("videoUrl");
+                    if (!"upsell".equals(e.getString("format"))) {
+                        int quality = Integer.parseInt((String)e.get("quality"));
+                        if (quality > bestQual) {
+                            bestQual = quality;
+                            vidUrl = (String)e.get("videoUrl");
+                        }
                     }
                 }
                 if (vidUrl == null) {

